4-WebApp-your-API/4-1-MyOrg/README.md

```diff
@@ -1,225 +1,256 @@
+   1. In the app's registration screen, select the **Expose an API** blade to the left to open the page where you can declare the parameters to expose this app as an API for which client applications can obtain [access tokens](https://docs.microsoft.com/azure/active-directory/develop/access-tokens) for.
+   The first thing that we need to do is to declare the unique [resource](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow) URI that the clients will be using to obtain access tokens for this API. To declare an resource URI, follow the following steps:
+      - Select `Set` next to the **Application ID URI** to generate a URI that is unique for this app.
+      - For this sample, accept the proposed Application ID URI (`api://{clientId}`) by selecting **Save**.
+   1. All APIs have to publish a minimum of one [scope](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow#request-an-authorization-code) for the client's to obtain an access token successfully. To publish a scope, follow these steps:
+         - Select **Add a scope** button open the **Add a scope** screen and Enter the values as indicated below:
+           - For **Scope name**, use `access_as_user`.
+           - Select **Admins and users** options for **Who can consent?**.
+           - For **Admin consent display name** type `Access TodoListService-aspnetcore-webapi`.
+           - For **Admin consent description** type `Allows the app to access TodoListService-aspnetcore-webapi as the signed-in user.`
+           - For **User consent display name** type `Access TodoListService-aspnetcore-webapi`.
+           - For **User consent description** type `Allow the application to access TodoListService-aspnetcore-webapi on your behalf.`
+           - Keep **State** as **Enabled**.
+           - Select the **Add scope** button on the bottom to save this scope.
+      1. Select the `Manifest` blade on the left.
+      - Set `accessTokenAcceptedVersion` property to **2**.
+      - Click on **Save**.
+   
+   #### Configure the service app (TodoListService-aspnetcore-webapi) to use your app registration
+
+   Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
+
+   > In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+   
+   1. Open the `TodoListService\appsettings.json` file.
+      1. Find the key `Domain` and replace the existing value with your Azure AD tenant name.
+      1. Find the key `TenantId` and replace the existing value with your Azure AD tenant ID.
+      1. Find the key `ClientId` and replace the existing value with the application ID (clientId) of `TodoListService-aspnetcore-webapi` app copied from the Azure portal.
+      
+   ### Register the client app (TodoListClient-aspnetcore-webapi)
+
+   **For more information, visit** [Register Application AAD](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app)
```

Comment:
>en-us/ [](http://example.com/codeflow?start=89&length=6)

check all code gen code and readme to ensure the language code (en-us/) is never printed anywhere